OpenSSL-Strategy.md

```diff
@@ -117,12 +117,55 @@ For Node.js >= 11.9.0:
 * FIPS: not supported
 
 
-### Node.js version 13.x
+Node.js EOL dates:
+- 12.x: April 2022
+- 13.x: June, 2020
+- 14.x: April 2023
 
-It's too early to discuss in detail, but this is the first version that may
-possibly support OpenSSL 3.0.0, and OpenSSL 3.0.0 is the next version of OpenSSL
-that is planned to support FIPS. See [OpenSSL 3.0.0 and FIPS][].
+## Node.js version 15.x (est. Oct 2020) (EOL Jun 2021)
 
+* OpenSSL version: 1.1.1
+* Allowed shared OpenSSL version: 1.1.1
+* Default minimum TLS version is TLSv1.2, default maximum is TLSv1.3. TLSv1
+  and TLSv1.1 are *not* supported by default, only by explicit run-time
+  configuration.
+* FIPS: not supported
+
+Note: OpenSSL 3.0.0 is NOT expected until Q4 2020, so unless they are _early_
+it won't happen in time for 15.x. Its possible that over the support time
+of 15.x (which is only about 8 months), that experimental support for OpenSSL
+3.x will be backported to it.
+
+## Node.js version 16.x (est Apr 2021) (EOL Apr 2024)
+
+* OpenSSL version: 3.x (because of OpenSSL EOL)
+* Allowed shared OpenSSL version: 1.1.1 (proposed...)
+* Default minimum TLS version is TLSv1.2, default maximum is TLSv1.3. TLSv1
+  and TLSv1.1 are *not* supported by default, only by explicit run-time
+  configuration.
+* FIPS: not supported
```

Comment:
or, maybe supported? it could happen.

Reply:
Current expectation, as I understand it, is that the FIPS provider *should* be available by the end of the year

Reply:
Even if it comes for a later version in the 3.x as long as it's not breaking we should be able to get to FIPs as well. Therefore, it is more of a "maybe" at this point.